game.js

```diff
@@ -0,0 +1,74 @@
+   var counter = 0;
+
+   var userName = prompt("Hello what's your name?");
+    console.log("This is the users name" + " " + userName);
+    alert("Hi" + " " + userName + " " + "I'm going to ask you a couple of yes or no questions so you can get to know me.");
+
+   var ques1 = prompt("Do I have any pets?");
+    console.log("Answer to ques1" + " " + ques1);
+
+   if (ques1 === "no" ||  ques1 === "NO"){
+    console.log("no is correct");
+    alert(userName + " " + "you're right I dont have any pets.");
+    counter ++;
+  } else if (ques1 === "yes" || ques1 === "YES") {
+      console.log("yes is incorrect");
+      alert("Wrong! I don't have any pets");
+   } else if (ques1 !== "yes" || "YES" || "no" || "NO"){
```

Comment:
This could just be an `else {}`, which would catch anything other than the Yes or No answers.